src/pages/settings/Wallet/PaymentMethodList.js

```diff
@@ -189,6 +193,18 @@ function PaymentMethodList(props) {
      */
     const renderListEmptyComponent = useCallback(() => <Text style={[styles.popoverMenuItem]}>{translate('paymentMethodList.addFirstPaymentMethod')}</Text>, [translate]);
 
+    const renderListFooterComponent = useCallback(
+        () => (
+            <MenuItem
+                onPress={props.onPress}
+                title={translate('walletPage.addBankAccount')}
+                icon={Expensicons.Plus}
+                wrapperStyle={[styles.cardMenuItem]}
```

Comment:
It could be an array and object as well.
```suggestion
                wrapperStyle={styles.cardMenuItem}
```